source/blog/2016-08-29-this-week-in-miq.html.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Last Week in ManageIQ: Speedups, Chargebacks, Toolbars, and more!"
+date: 2016-08-29
+author: jntullo
+comments: true
+published: true
+---
+
+[Jillian](github.com/jntullo) here to guide you through another edition of Last Week in ManageIQ. Let's get this show on the road and explore what we've accomplished this week!
+
+## Featured
+As usual, we had a great week in ManageIQ with 94 pull requests merged from our awesome [contributors](https://github.com/ManageIQ/manageiq/graphs/contributors). Next week, one of them could be you! 
+Head on over to our [issues](https://github.com/manageiq/manageiq/issues) to see where you can contribute - we'd love to have you.
+
+## Improved
+
+### [Index Speedup](https://github.com/ManageIQ/manageiq/pull/10685)
+The _VMs & Instances in My LDAP Group_ index and ajax calls in `/vm_or_template` received an awesome speedup this week
```

Comment:
@jntullo - Possibly can we remove the `The` at the beginning and prefix ajax with an adjective?  I really like the sentence but I'm getting tongue twisted in reading the multiple menu names. 😄